versions.tf

```
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 5.3, < 7"
+      version = "> 5.39, < 7"
```

Looks like this was added **in** 5.39, so >= should be fine.

```suggestion
      version = ">= 5.39, < 7"
```